internals_fsutils.go

```diff
@@ -382,53 +386,51 @@ func createZip(archiveName string, files map[string][]byte) error {
 		if err != nil {
 			return err
 		}
-		_, err = f.Write([]byte(fcont))
+		_, err = io.Copy(f, fcont)
```

Comment:
I suggest renaming `fcont` to `freader` to be consistent with https://github.com/jonsyu1/seelog/pull/3/files#diff-e365fbb09f8d29dc59e3030c8cb3462cR406